index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0"> <title>Testing</title>
+</head>
+<body>
+  <h1>Hello World!</h1>
+  <p>this is a test file for pull request</p>
+  <h1>My name is Sir Lance Valerio, nice to meet you!</h1>
```

Comment:
Please change this code:
```suggestion
  <p>My name is Sir Lance Valerio, nice to meet you!</p>
```